cetmix_tower_server/wizards/cx_tower_server_template_create_wizard.py

```diff
@@ -47,7 +48,52 @@ class CxTowerServerTemplateCreateWizard(models.TransientModel):
     line_ids = fields.One2many(
         comodel_name="cx.tower.server.template.create.wizard.line",
         inverse_name="wizard_id",
+        string="Configuration Variables",
     )
+    required = fields.Boolean(
+        string="Required", help="Indicates if this configuration parameter is mandatory"
+    )
```

Comment:
The required field at the wizard level was indeed redundant and misplaced. Since the requirement flag is a property of individual configuration variables (handled at the line model level), it makes sense to remove it from the wizard class.

The field removal has no impact on functionality as the requirement validation already occurs on the line_ids level.

Resolution:
The redundant required field has been removed from the wizard class. The line model correctly handles the "required" logic as intended.